pkg/services/accesscontrol/eval/scope.go

```diff
@@ -0,0 +1,26 @@
+package eval
+
+import (
+	"fmt"
+	"strings"
+)
+
+const (
+	ScopeNone = ""
```

Comment:
This name carries the wrong semantics, where it's used it's used to match any or no scope for the permission ("the scopes are irrelevant").

Ideally, I'd want us to not have it at all (just checking for the permission is and will remain a fairly common operation, especially when we start digging a bit deeper) and keep the variadic behavior that was in place previously (`Permission(action string, scope ...string)` with zero or one scope).